sources/packages/web/src/views/aest/institution/ApproveDenyDesignation.vue

```diff
@@ -1,33 +1,75 @@
 <template>
-  <!-- CSS class temporary-modal is work around for lack of responsiveness of v-dialog. -->
-  <modal-dialog-base
-    :showDialog="showDialog"
-    @dialogClosed="dialogClosed"
-    :title="title"
-  >
-    <template v-slot:content>
-      <div class="temporary-modal">
-        <formio
-          formName="approvedenydesignation"
-          :data="designation"
-          @loaded="formLoaded"
-          @submitted="submitDesignationUpdate"
-        ></formio>
-      </div>
-    </template>
-    <template v-slot:footer>
-      <check-permission-role :role="Role.InstitutionApproveDeclineDesignation">
-        <template #="{ notAllowed }">
+  <v-form ref="approveDenyDesignation">
```

Comment:
On the screenshot, it seems that there is some content being cut and no scrollbar. Is it right?
![image](https://user-images.githubusercontent.com/61259237/197845196-e3b64265-8f29-4af0-befd-28ff1003afbd.png)